catalog/to-consul/resource.go

```diff
@@ -6,7 +6,7 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/deckarep/golang-set"
+	mapset "github.com/deckarep/golang-set"
```

Comment:
I'm curious why you're renaming this?

Reply:
visual studio linter is doing this... i can revert if you want @ishustava

Reply:
Ah ok, no need to remove it. I was just curious 😃